_scripts/pm2-all.sh

```diff
@@ -11,6 +11,14 @@ fi
 
 mkdir -p artifacts
 
+echo "build shared fxa services..."
+yarn workspaces foreach \
+    --verbose \
+    --include fxa-auth-client \
+    --include fxa-react \
+    --include fxa-shared \
+    run build > artifacts/build-shared.log;
```

Comment:
Might want to put this in an `if..else` block? Otherwise if this fails, it doesn't have a message logged pointing to `artifacts/build-shared.log`.